frontend/src/pages/modelRegistry/screens/ModelVersions/ModelVersionListView.tsx

```diff
@@ -55,8 +58,18 @@ const ModelVersionListView: React.FC<ModelVersionListViewProps> = ({
     React.useState(false);
 
   const filteredModelVersions = filterModelVersions(unfilteredModelVersions, search, searchType);
+  const date = rm?.lastUpdateTimeSinceEpoch && new Date(parseInt(rm.lastUpdateTimeSinceEpoch));
 
   if (unfilteredModelVersions.length === 0) {
+    if (isArchiveModel) {
+      return (
+        <EmptyModelRegistryState
+          testid="empty-archive-model-versions"
+          title="No versions"
+          description={`${rm?.name} has no registered versions.`}
+        />
+      );
+    }
```

Comment:
@mturley @yih-wang I am not sure, should we have Empty state for versions of Archived models? for now, I have just re-used the one - empty state for versions by removing the primary action. Let me know your thoughts on this?

Reply:
Had discussion with @yih-wang and I will update the Empty state for version with archived model.